gprofiler/__init__.py

```diff
@@ -1 +1,5 @@
-__version__ = "0.0.7"
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+__version__ = "0.0.8"
```

Comment:
We will release the next version probably before merging this PR, so you can drop it. Also, I think it's better in general if we bump the version in separate PRs (because after merging this & before releasing a new version, we will want to run some extra testing on the version before publishing it)